schedule/yast/sle/guided_xfs/guided_xfs.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name:           guided_ext4
+description:    >
+  Guided Partitioning installation with xfs filesystem.
+vars:
+  YUI_REST_API: 1
+schedule:
+  guided_filesystem:
+    - installation/partitioning/guided_setup/select_filesystem_option_xfs
+  default_systemd_target:
+    - installation/installation_settings/validate_default_target
+  system_validation:
+    - console/validate_partition_table_via_blkid
+    - console/validate_blockdevices
+    - console/validate_free_space
+test_data:
+  <<: !include test_data/yast/xfs/xfs_partition.yaml
```

Comment:
as in this test suite we don't have test data in the test suite directly, could we please add this files to YAML_TEST_DATA so we can reduce the number of files already (I know if it is not much gain, as the validation are not the same, they will be in the future, but it would be good to give that little step to set an example) wdyt?

Ahm ok, I see you can move test data here, but it should be the opposite, leave them in Job group would allow us to have only one file here.